python/solutii/bogdan_iacoboae/icao/from_ICAO/decodat.icao

```diff
@@ -0,0 +1,9 @@
+mesaj
```

Comment:
Am facut acest fisier pentru ca mesajul decodat sa fie scris si intr-un alt fisier in afara de output-ul de pe ecran.